### PR TITLE
possibly python3 version of popcorn

### PR DESCRIPTION
**Pull request by shafayetrahat**
Please check the code change and see if the changes are ok.

### CODE REVIEW

**Comment by shafayetrahat** [comment]
@brielin can you please check the changes if they reproduce the same result?

**Comment by brielin** [comment]
Hi I'm at a conference at the moment but I can try to take a look at this
in the next week or so.

Did you check the included test files already?

**Comment by shafayetrahat** [comment]
Yes, I have. The output produces smoothly. However, as I'm unable to
reproduce the output result with python 2, I'm not sure if the output
are in the right format or not. Also, I've made some small changes
mentioned in the issues. I'm not sure if they have an effect on the algorithm
of the code or not. Also, the popcorn module after installation doesn't
work due to some dependency issue. So, I've used __main__.py in the
test.bash file.

**Comment by shafayetrahat** [comment]
popcorn binary command is working now

**Comment by brielin** [comment]
Hi - okay I checked this all on my machine and it looks great, nice work and thanks for the contribution. I did try to port this to Python3 some years ago, figuring it would be easy, but pysnptools didn't have a Python3 version at the time and I didn't have time to write a custom bed processor in python 3. Glad to see that is updated now.

What do you mean "unable to reproduce the output result"? I'm seeing effectively the same numbers, although there are very tiny differences in the MLE SEs, I'm guessing there have been small changes in the optimization defaults in the dependent packages over the years. It doesn't look to me that your changes have changed the algorithm or analysis SNPs in any way.

Will wait for your reply to close and merge.

File: popcorn/__main__.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
 from __future__ import division
```
Comment (brielin) [inline popcorn/__main__.py]: These from __future__ imports aren't needed in python 3 so you should remove them in this PR.

Comment (shafayetrahat) [reply popcorn/__main__.py]: @brielin Removed the unused import from the future library.

**Comment by shafayetrahat** [comment]
@brielin 

popcorn fit -v 1 --use_mle --cfile EUR_EAS_test_ge.cscore --gen_effect --sfile1 EUR_test.txt --sfile2 EAS_test.txt EAS_EUR_test_ge_result.txt

gives error like

Beginning analysis at Sat May 21 07:44:33 2022
Analyzing a pair of traits in two populations
49999 49999 SNPs detected in input.
43897 SNPs remaining after merging with the score file.
43897 SNPs remaining after filtering on AF and self-compliment SNPs.
Initial estimate:
       Val (obs)  SE
h1^2   0.172133 NaN
h2^2   0.086708 NaN
pg     0.302370 NaN
Performing jackknife estimation of the standard error using 200 blocks of size 219 .
**/home/shafayetrahat/t2d/venv/lib/python3.9/site-packages/pandas/core/arraylike.py:397: RuntimeWarning: invalid value encountered in log
  result = getattr(ufunc, method)(*inputs, **kwargs)
/home/shafayetrahat/t2d/venv/lib/python3.9/site-packages/pandas/core/arraylike.py:397: RuntimeWarning: invalid value encountered in log
  result = getattr(ufunc, method)(*inputs, **kwargs)
/home/shafayetrahat/t2d/venv/lib/python3.9/site-packages/pandas/core/arraylike.py:397: RuntimeWarning: invalid value encountered in log**
  result = getattr(ufunc, method)(*inputs, **kwargs)
Jackknife iter: 200 
      Val (obs)        SE          Z  P (Z)
h1^2   0.172133  0.012872  13.372961    0.0
h2^2   0.086708  0.008554  10.136322    0.0
pge    0.302370  0.053021  13.157684    0.0
Analysis finished at Sat May 21 07:45:29 2022

**Although it finishes analysis and produces the EAS_EUR_test_ge_result.txt file. The error confused me**

**Comment by shafayetrahat** [comment]
@brielin  If you think these are minor problems please merge my request

**Comment by shafayetrahat** [comment]
Any update?

**Comment by brielin** [comment]
yeah, that log warning appeared some years ago and has been there. it doesn't seem to effect the results but I haven't dug into what is generating it. either way, it doesn't show in the default regression setting so I think this is fine.

**Comment by shafayetrahat** [comment]
Awesome, thanks! However, plz change the MD file to your liking as now
popcorn supports python 3

**Comment by brielin** [comment]
Done!

**Pull request merged**